packages/ListBox/stories/examples/multi.js

```diff
@@ -1,257 +0,0 @@
-import React from "react";
-import tokens from "@paprika/tokens";
-import { css } from "styled-components";
-import ListBox from "../../src";
-import * as characters from "../fixtures/characters";
-
-export const BasicWithPreselectedOptions = () => (
-  <ListBox isMulti isOpen>
-    <ListBox.Option isSelected>Venom</ListBox.Option>
-    <ListBox.Option isSelected>Thunderbolts</ListBox.Option>
-    {characters.villians}
-    <ListBox.Option isSelected>Wolverine</ListBox.Option>
-  </ListBox>
-);
-
-function OptionCustomCheckbox(props) {
-  return (
-    <span>
-      <input tabIndex="-1" type="checkbox" checked={props.isSelected} />
-      {props.character}
-    </span>
-  );
-}
-
-export const WithCustomCheckboxes = () => (
-  <ListBox isMulti>
-    {characters.villians.map(Option => (
-      <ListBox.Option label={Option.props.children}>
-        {({ isSelected }) => <OptionCustomCheckbox isSelected={isSelected} character={Option.props.children} />}
-      </ListBox.Option>
-    ))}
-  </ListBox>
-);
-
-export const WithGroups = () => (
```

Comment:
We need to preserve this example – but in the Docs page is probably a better place for it.  There's not currently an example of groups there though, so I'm worried we'll forget if we don't keep anything in this PR.